pycode/memilio-epidata/memilio/epidata/getPopulationData.py

```diff
@@ -283,45 +284,27 @@ def test_total_population(df_pop, age_cols):
         raise gd.DataError('Total Population does not match expectation.')
 
 
-def get_population_data(read_data=dd.defaultDict['read_data'],
-                        file_format=dd.defaultDict['file_format'],
-                        out_folder=dd.defaultDict['out_folder'],
-                        merge_eisenach=True,
-                        username='',
-                        password='',
-                        **kwargs):
-    """! Download age-stratified population data for the German counties.
-
-    The data we use is:
-    Official 'Bevölkerungsfortschreibung' 12411-02-03-4:
-    'Bevölkerung nach Geschlecht und Altersgruppen (17)' 
-    of regionalstatistik.de. 
-    ATTENTION: The raw file cannot be downloaded 
-    automatically by our scripts without an Genesis Online account. In order to
-    work on this dataset, please enter your username and password or manually download it from:
-
-    https://www.regionalstatistik.de/genesis/online -> "1: Gebiet, Bevölkerung,
-    Arbeitsmarkt, Wahlen" -> "12: Bevölkerung" -> "12411 Fortschreibung des
-    Bevölkerungsstandes" ->  "12411-02-03-4: Bevölkerung nach Geschlecht und 
-    Altersgruppen (17) - Stichtag 31.12. - regionale Tiefe: Kreise und
-    krfr. Städte". 
-
-    Download the xlsx or csv file and put it under dd.defaultDict['out_folder'], 
-    this normally is Memilio/data/pydata/Germany. 
-    The folders 'pydata/Germany' have to be created if they do not exist yet. 
-    Then this script can be run.
+def fetch_population_data(read_data: bool = dd.defaultDict['read_data'],
+                          out_folder: str = dd.defaultDict['out_folder'],
+                          username='',
+                          password='',
+                          **kwargs
+                          ) -> pd.DataFrame:
+    """! Downloads or reads the population data.
+    Available data starts from 2020-04-24.
```

Comment:
Population Data does not have a start_date.

Reply:
True!